<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>illimunationDialog</class>
 <widget class="QWidget" name="illimunationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="horizontalFrame_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;Click to illuminate the dome!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRed">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>R</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
     border-style: inset;
     border-width: 1px;
     border-radius:5px;
	 padding: 3px;
     border-style: solid;
     border-width: 1px;
     border-radius:5px;
     border-color: palette(button);

    background-color: qradialgradient(cx: 0.7, cy: 1.4, fx: 0.7, fy: 1.9, radius: 1.5, stop: 0 rgb(255,0, 0), stop: 1 transparent );
	 padding: 3px;
     font-size: 10pt;
}

QPushButton:checked { 
     border-color: palette(dark);
     background-color: rgb(255, 0, 0);

}

QPushButton:pressed { 
     border-color: palette(dark);
     background-color: qradialgradient(cx: 0.7, cy: 1.4, fx: 0.7, fy: 1.9, radius: 1.5, stop: 0 palette(dark), stop: 1 transparent );
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGreen">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>G</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
     border-style: inset;
     border-width: 1px;
     border-radius:5px;
	 padding: 3px;
     border-style: solid;
     border-width: 1px;
     border-radius:5px;
     border-color: palette(button);

    background-color: qradialgradient(cx: 0.7, cy: 1.4, fx: 0.7, fy: 1.9, radius: 1.5, stop: 0 rgb(0,255, 0), stop: 1 transparent );
	 padding: 3px;
     font-size: 10pt;
}

QPushButton:checked { 
     border-color: palette(dark);
background-color: rgb(0, 255, 0);

}

QPushButton:pressed { 
     border-color: palette(dark);
     background-color: qradialgradient(cx: 0.7, cy: 1.4, fx: 0.7, fy: 1.9, radius: 1.5, stop: 0 palette(dark), stop: 1 transparent );
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBlue">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>B</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton { 
     border-style: inset;
     border-width: 1px;
     border-radius:5px;
	 padding: 3px;
     border-style: solid;
     border-width: 1px;
     border-radius:5px;
     border-color: palette(button);

 background-color: qradialgradient(cx: 0.7, cy: 1.4, fx: 0.7, fy: 1.9, radius: 1.5, stop: 0 rgb(0,0, 255), stop: 1 transparent );
	 padding: 3px;
     font-size: 10pt;
}

QPushButton:checked { 
     border-color: palette(dark);
background-color: rgb(0, 0, 255);
   
}

QPushButton:pressed { 
     border-color: palette(dark);
     background-color: qradialgradient(cx: 0.7, cy: 1.4, fx: 0.7, fy: 1.9, radius: 1.5, stop: 0 palette(dark), stop: 1 transparent );
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWhite">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>W</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
     border-style: inset;
     border-width: 1px;
     border-radius:5px;
	 padding: 3px;
     border-style: solid;
     border-width: 1px;
     border-radius:5px;
     border-color: palette(button);
    background-color: qradialgradient(cx: 0.7, cy: 1.4, fx: 0.7, fy: 1.9, radius: 1.5, stop: 0 rgb(255,255, 255), stop: 1 transparent );
	 padding: 3px;
     font-size: 10pt;
}

QPushButton:checked { 
     border-color: palette(dark);
background-color: rgb(255, 255, 255);
}

QPushButton:pressed { 
     border-color: palette(dark);
     background-color: qradialgradient(cx: 0.7, cy: 1.4, fx: 0.7, fy: 1.9, radius: 1.5, stop: 0 palette(dark), stop: 1 transparent );
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
